zip-XXXX.rst

```diff
@@ -0,0 +1,451 @@
+with the existing Zcash clients.
+
+Though the Foundation shall periodically judge the efficacy of dev fee
+recipients, deciding on and driving roadmaps for Zcash is the role of dev fee
+recipients, in partnership with the community. This role is neatly balanced by
+users running full nodes and miners, either of which can veto a network upgrade.
+
+While dev fee recipients are not required to work exclusively on Zcash,
+considering the nature of their work, recipients must guarantee they aren't
+obliged to the interests of competing projects.
+
+The role of the principal developer
+-----------------------------------
+
+The role of the principal developer is as a "first among equals" amongst the dev
+fee recipients.
+
+The principal developer shall make a number of guarantees.
+
+1. Zcash shall be their exclusive focus, submitting financials periodically to
+   the Foundation as assurance.
+2. They shall maintain a well-run board and employ a qualified CFO.
+3. In addition to the existing open-source requirements, they shall agree to
+   assign any trademarks or patents relevant to Zcash to the Foundation.
+
+In exchange, the principal developer is granted an indefinite dev fee allocation
+and a wide remit to pursue longer-term research relevant to Zcash, as well as
+a voice on the board of the Foundation. The principal developer will not be
+subject to semi-annual review, though it will submit the same plans required of
+other recipients. The principal developer will only be recommended for removal
+by the Foundation in extraordinary circumstances, including reneging on the
+above guarantees or extreme negligence.
+
+Minimum viable Foundation
+-------------------------
+
+To manage the dev fee and fulfill its community and diligence duties, the
+Foundation shall maintain a board of 5 independent members. Rather than the
+structure in the current bylaws, the board will consist of
+
+* 1 seat voted on by ZEC holders directly.
+* 1 seat representing a newly created research advisory board, whose primary
+  role will be technical diligence of potential recipients of the dev fee.
+* 1 seat for the "principal developer", a privileged recipient of the Zcash
+  dev fee acting as "first among equals" amongst a variety of dev fee recipients
+  building on Zcash.
+* 2 seats elected by the board, as the board is currently selected according to
+  the bylaws. The board's discretion here means these could be selected via a
+  community election, or via the remaining 3 seats' direct vote.
+
+The Foundation requires a professional board. Board member selection should
+heavily favor candidates with existing formal public or private sector board
+experience. Each board member shall be paid reasonably by the Foundation.
+
+Each board member should bring a unique network and set of skills to bear to
+increase the impact of the Foundation.
+
+Outside the seat for the principal developer, no board members shall have an
```

Comment:
Since the whole point of this system is to decentralized the Dev Fee and have it support numerous players in the ecosystem, directly and the grants, we may reach a point where most of the people who are technically cognizant and aligned with Zcash's mission will be tainted by Dev Fund. (Arguably, this is already the case!) So your requirement here effectively excludes BUIDLers from the board, leaving it just to financial interests and outsiders.

Reply:
I don't know, between academia and holders I think there should be plenty of people with interest and skin in the game?

I'm also loosely assuming fewer than 10 recipients of the fee for the medium-term. Whether that's wise is worth debate, but I'm looking to engender a situation similar to Ethereum's multi client implementation teams